README.rst

```diff
@@ -234,6 +234,18 @@ As of v0.4.0 you can toggle the maintenance mode on a host.
             host.set_maintenance(True)
 
 
+**Caching**
+
+As of v0.7.0 the use of requests_cache has been dropped in favor of an internal
+cache that stores the responses from a subset of the single-item endpoint
+methods. A future goal is to allow more granular control over the caching policy,
+but for now the following endpoint responses are cached:
```

Comment:
Yes, per-process, in-memory cache. In an ideal world a user could provide their own caching by passing in something that follows the basic get/set api, but I can't get motivated for that while I'm pretty much still the only user (AFAIK).